tools/topology/topology2/sof-ace-tplg/tplg-targets.cmake

```diff
@@ -104,4 +104,8 @@ DEEPBUFFER_FW_DMA_MS=100,DEEPBUFFER_D0I3_COMPATIBLE=true"
 
 "cavs-sdw\;sof-lnl-rt711-l0-rt1316-l23-rt714-l1\;PLATFORM=lnl,NUM_SDW_AMP_LINKS=2,SDW_DMIC=1,\
 NUM_HDMIS=0,SDW_SPK_STREAM=SDW2-Playback,SDW_SPK_IN_STREAM=SDW2-Capture,SDW_DMIC_STREAM=SDW1-Capture"
+
+"cavs-sdw\;sof-lnl-rt712-l3-rt1712-l1\;PLATFORM=lnl,SDW_DMIC=1,NUM_HDMIS=0,NUM_SDW_AMP_LINKS=1,\
+SDW_AMP_FEEDBACK=false,SDW_SPK_STREAM=Playback-SmartAmp,SDW_DMIC_STREAM=Capture-SmartMic,\
```

Comment:
paranoia check: are you sure there's no amp feedback? all recent amps have such a stream.

Reply:
The codec spec says amp feedback is supported, but there is no feedback DAI declared in codec driver.

Reply:
ok, thanks @aiChaoSONG 

We'll probably need an update at some point but there's plenty of time to deal with this later.